pkgs/os-specific/linux/musl/default.nix

```diff
@@ -62,6 +62,8 @@ stdenv.mkDerivation rec {
       url = "https://raw.githubusercontent.com/openwrt/openwrt/87606e25afac6776d1bbc67ed284434ec5a832b4/toolchain/musl/patches/300-relative.patch";
       sha256 = "0hfadrycb60sm6hb6by4ycgaqc9sgrhh42k39v8xpmcvdzxrsq2n";
     })
+    # wcsnrtombs destination buffer overflow, remove >= 1.2.2
+    ./CVE-2020-28928.patch
```

Comment:
Why not fetchpatch https://www.openwall.com/lists/oss-security/2020/11/20/4/1 directly?

Reply:
No reason in particular. Will swap it out.

Reply:
Used fetchurl, as it was already imported.